formspree/forms/models.py

```diff
@@ -42,6 +60,16 @@ def __init__(self, email, host):
     def __repr__(self):
         return '<Form %s, email=%s, host=%s>' % (self.id, self.email, self.host)
 
+    def get_random_like_string(self):
+        if not self.id:
+            raise Exception("this form doesn't have an id yet, commit it first.")
+        return int2bigstring(self.id)
```

Comment:
Yeah I'm happy with it. Generally if a primary key is sequential I try to not include it in a URL, but it should be fine. 

//Cole